lib/node_modules/@stdlib/blas/base/drot/README.md

```diff
@@ -0,0 +1,216 @@
+-   **x**: first input [`Float64Array`][mdn-float64array].
+-   **strideX**: index increment for `x`.
+-   **y**: second input [`Float64Array`][mdn-float64array].
+-   **strideY**: index increment for `y`.
+-   **c**: cosine of the angle of rotation.
+-   **s**: sine of the angle of rotation.
+
+The `N` and stride parameters determine how values in the strided arrays are accessed at runtime. For example, to apply plane rotation to elements in the strided arrays in reverse order
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+
+var x = new Float64Array( [ 1.0, 2.0, 3.0, 4.0, 5.0, 6.0 ] );
+var y = new Float64Array( [ 7.0, 8.0, 9.0, 10.0, 11.0, 12.0 ] );
+
+drot( x.length, x, 2, y, 2, 0.707, 0.707 );
+// x => <Float64Array>[ 5.656, 2.0, 8.484, 4.0, 11.312, 6.0 ]
+// y => <Float64Array>[ 4.242, 8.0, ~4.242, 10.0, ~4.242, 12.0 ]
+```
+
+Note that indexing is relative to the first index. To introduce an offset, use [`typed array`][mdn-typed-array] views.
+
+<!-- eslint-disable stdlib/capitalized-comments -->
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+
+// Initial arrays...
+var x0 = new Float64Array( [ 1.0, 2.0, 3.0, 4.0, 5.0, 6.0 ] );
+var y0 = new Float64Array( [ 7.0, 8.0, 9.0, 10.0, 11.0, 12.0 ] );
+
+// Create offset views...
+var x1 = new Float64Array( x0.buffer, x0.BYTES_PER_ELEMENT*1 ); // start at 2nd element
+var y1 = new Float64Array( y0.buffer, y0.BYTES_PER_ELEMENT*3 ); // start at 4th element
+
+drot( 3, x1, -2, y1, 1, 0.707, 0.707 );
+// x0 => <Float64Array>[ 1.0, 9.898, 3.0, 10.604999999999999, 5.0, 11.312 ]
+// y0 => <Float64Array>[ 7.0, 8.0, 9.0, 2.8279999999999994, 4.949, 7.07 ]
+```
+
+#### drot.ndarray( N, x, strideX, offsetX, y, strideY, offsetY, c, s )
+
+Applies a plane rotation using alternative indexing semantics.
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+
+var x = new Float64Array( [ 1.0, 2.0, 3.0, 4.0, 5.0 ] );
+var y = new Float64Array( [ 6.0, 7.0, 8.0, 9.0, 10.0 ] );
+
+drot.ndarray( x.length, x, 1, 0, y, 1, 0, 1.0, 0.0 );
+// x => <Float64Array>[ 1.0, 2.0, 3.0, 4.0, 5.0 ]
+// y => <Float64Array>[ 6.0, 7.0, 8.0, 9.0, 10.0 ]
+```
+
+The function has the following additional parameters:
+
+-   **offsetX**: starting index for `x`.
+-   **offsetY**: starting index for `y`.
+
+While [`typed array`][mdn-typed-array] views mandate a view offset based on the underlying buffer, the offset parameters support indexing semantics based on starting indices. For example, to apply rotation for every other value in `x` starting from second value,...,
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+
+var x = new Float64Array( [ 1.0, 2.0, 3.0, 4.0, 5.0, 6.0 ] );
+var y = new Float64Array( [ 7.0, 8.0, 9.0, 10.0, 11.0, 12.0 ] );
+
+drot.ndarray( 4, x, 0, 2, y, 0, 2, 0.707, 0.707 );
+// x => <Float64Array>[ 1.0, 2.0, -2.998188273612, 4.0, 5.0, 6.0 ]
+// y => <Float64Array>[ 7.0, 8.0, -8.994564820835999, 10.0, 11.0, 12.0 ]
```

Comment:
Is this right? Of the values on which the operation applies, only a couple of values are updated? If so, we should update `c` and `s` to something more "interesting"; otherwise, it is confusing to readers as they need to determine why the values are the same. Was it because of `N` or because of `c` and `s` or because of the nature of the affected values, etc etc?